cspatients/views.py

```diff
@@ -185,6 +185,42 @@ def post(self, request):
         return Response(data, status=return_status)
 
 
+class MultiSelectView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        valid = True
+        selected_items = []
+
+        def clean_and_split(str):
+            return [x.strip() for x in str.split(",") if x]
+
+        def is_int(s):
+            try:
+                int(s)
+                return True
+            except ValueError:
+                return False
+
+        selections = clean_and_split(request.GET.get("selections", ""))
+        options = clean_and_split(request.GET.get("options", ""))
+
+        for item in selections:
+            if not is_int(item):
```

Comment:
```suggestion
            if type(item) is not int:
```

Reply:
Not really the same thing, my check is `can this string be converted to int` instead of `is this var a int`